main.go

```diff
@@ -112,7 +114,15 @@ var (
 	seed []byte
 )
 
-const garbleMapFile = "garble.map"
+const (
```

Comment:
Put the constants only used by the checkGoVersion inside the checkGoVersion function.